src/quo2/components/numbered_keyboard/keyboard_key/style.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.style)
+
+(def container
+  {:width           48
+   :height          48
+   :justify-content :center
+   :align-items     :center
+   :border-radius   999})
+
+(defn text
+  [color]
+  {:color     color
+   :font-size 27})
+
+(defn icon
+  [color]
+  {:color  color
+   :width  30
```

Comment:
we don't have any icons that are size "30" and in the component it's `20` which is the default

Reply:
Done